src/scripts/main.js

```diff
@@ -1 +1,25 @@
 'use strict';
+
+document.addEventListener('DOMContentLoaded', () => {
+  const thumbsContainer = document.getElementById('thumbs');
+  const largeImg = document.getElementById('largeImg');
+
+  thumbsContainer.addEventListener('click', function (even) {
```

Comment:
There is a typo in the parameter name 'even'. It should be 'event' to correctly reference the event object.